packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/TestData/Statements/SwitchCaseStatementWithUsing.txt

```diff
@@ -0,0 +1,22 @@
+// License string
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.IO;
+
+public bool Foo()
+{
+    bool foo = true;
+    switch (foo)
+    {
+        case true:
+            using (var x = new global::System.IO.MemoryStream())
```

Comment:
should not be seeing MemoryStream here, you might need to add metatdata reference for typeof(object)

Reply:
That happen during the Roslyn processing right? These unit tests just test writing the statements, so there is no workspace setup or file generation called. I manually wrote these test data files, and asserted that the statements matched what I had written.

Reply:
ah, thats true make sense now.